javagen/src/main/java/com/azure/autorest/util/SchemaUtil.java

```diff
@@ -78,6 +78,9 @@ public static Schema getLowestCommonParent(List<Schema> schemas) {
         return chain.isEmpty() ? new AnySchema() : chain.getLast();
     }
 
+    /*
+     * Note: returns PrimitiveType.Void for operation that contains binary response body schema, can be checked by SchemaUtil.containsBinaryResponse(operation)
```

Comment:
Does this happen for all, or just DPG?

Should we instead make this method always return `ClassType.StreamResponse` for binary. Or a step further, make it return the correct response type, based on whether it is DPG (anyway, this is a utility method, won't hurt to make it do some branching)?
(so that we can remove some duplicate code in 2 Mapper)

Reply:
This happens for all.

Determining the correct response type seems like the way to go. 
 * `Flux<ByteBuffer>` for mgmt
 * `BinaryData` for DPG

Reply:
Probably:

vanillar/mgmt: InputStream
dpg: BinaryData

Note this is sync type, `Flux` is async type.